logger/Cargo.toml

```diff
@@ -8,23 +8,23 @@ authors.workspace = true
 license.workspace = true
 
 [dependencies]
-iroha_config = { version = "=2.0.0-pre-rc.16", path = "../config" }
+iroha_config = { workspace = true }
 
-color-eyre = "0.6.2"
-serde_json = "1.0.91"
-tracing = "0.1.37"
+color-eyre = { workspace = true }
+serde_json = { workspace = true }
+tracing = { workspace = true }
 tracing-core = "0.1.30"
 tracing-futures = { version = "0.2.5", default-features = false, features = ["std-future", "std"] }
```

Comment:
Shouldn't it be moved as well?

AFAIU if `tracing` is specified there, these related crates might be moved there as well.

Reply:
Yeah, probably it makes sense to move co-dependent crates